src/Logging/Scrubbing/RegexLogScrubbingRule.cs

```diff
@@ -24,12 +25,27 @@ public RegexLogScrubbingRule(string regexToReplace, string replacementValue)
 			m_replacementValue = replacementValue;
 		}
 
+		/// <summary>
+		/// Initializes a new instance of the <see cref="RegexLogScrubbingRule"/> class.
+		/// </summary>
+		/// <param name="regexToReplace">The regular expression specifying the strings to replace.</param>
+		/// <param name="matchEvaluator">Custom logic for regex replace.</param>
```

Comment:
Suggest adding a remarks tag detailing that this comes with an increased perf overhead.